bridge/client/app/_services/api.service.ts

```diff
@@ -114,6 +115,11 @@ export class ApiService {
     return this.http.get<UniformRegistration[]>(url);
   }
 
+  public getUniformRegistrationLogs(uniformRegistrationId: string, pageSize: number = 100): Observable<UniformRegistrationLog[]> {
+    const url = `${this._baseUrl}/controlPlane/v1/log?integrationId=${uniformRegistrationId}&pageSize=${pageSize}`;
+    return this.http.get<UniformRegistrationLog[]>(url);
```

Comment:
Result is an object not an array.
Logs are found in `.logs`

Reply:
Yup, already fixed this :)